### ISSUE TITLE
[Accessibility] Background color of onboarding screens is inaccessible

### ISSUE THREAD

**Issue by glenn-sorrentino** [guardianproject/haven#224]
The blue text color (#BBDEFB) doesn’t have enough contrast against the blue background color (#1976D2) to make the app unusable for some users with color blindness. It can cause strain on the eyes of users with good vision, and block users with poor vision entirely.

WCAG 2.0 level AA recommends a contrast ratio of 4.5:1 for normal text and 3:1 for large text. This is a less strict guideline and helps to ensure that people with color blindness or poor vision can read the text on the screen.

You can check colors to see if they pass here: https://webaim.org/resources/contrastchecker/

Current Background Color:
![tour - page 2 - current](https://user-images.githubusercontent.com/28545431/35116114-2170c8b2-fc3f-11e7-94a5-4b8772cc52f0.png)

Accessible Background Color:
![tour - page 2 - accessible colors](https://user-images.githubusercontent.com/28545431/35116120-286e13a4-fc3f-11e7-84f3-3d1ed96c7ce7.png)

Pages from [design audit](https://github.com/guardianproject/haven/issues/218):

[Tour - Page 2.pdf](https://github.com/guardianproject/haven/files/1643945/Tour.-.Page.2.pdf)
![tour - page 2](https://user-images.githubusercontent.com/28545431/35114072-667047e6-fc38-11e7-859d-3d4d70147002.jpg)

[Tour - Page 2 Current Colors.pdf](https://github.com/guardianproject/haven/files/1643946/Tour.-.Page.2.Current.Colors.pdf)
![tour - page 2 - current colors](https://user-images.githubusercontent.com/28545431/35114084-6cc89c4c-fc38-11e7-8e21-b3decf526da9.jpg)

[Tour - Page 2 Accessible Colors.pdf](https://github.com/guardianproject/haven/files/1643947/Tour.-.Page.2.Accessible.Colors.pdf)
![tour - page 2 - accessible colors](https://user-images.githubusercontent.com/28545431/35114091-755638d8-fc38-11e7-9c2a-79dccb9f60aa.jpg)
